templates/web-deployment.yaml

```
@@ -263,6 +283,10 @@ spec:
             - name: CONCOURSE_RESOURCE_CHECKING_INTERVAL
               value: {{ .Values.concourse.web.resourceCheckingInterval | quote }}
             {{- end }}
+            {{- if .Values.concourse.web.resourceWithWebhookCheckingInterval }}
+            - name: CONCOURSE_RESOURCE_WITH_WEBHOOK_CHECKING_INTERVAL
+              value: {{ .Values.concourse.web.resourceCheckingInterval | quote }}
```

looks like copy-pasted from the above lines; missed a spot 🙂 
```suggestion
              value: {{ .Values.concourse.web.resourceWithWebhookCheckingInterval | quote }}
```